pandas/tests/arithmetic/test_numeric.py

```diff
@@ -42,6 +42,30 @@ def test_operator_series_comparison_zerorank(self):
         expected = 0.0 > pd.Series([1, 2, 3])
         tm.assert_series_equal(result, expected)
 
+    def test_df_numeric_cmp_dt64_raises(self):
+        # GH#8932, GH#22163
+        ts = pd.Timestamp.now()
+        df = pd.DataFrame({'x': range(5)})
+        with pytest.raises(TypeError):
+            df > ts
+        with pytest.raises(TypeError):
+            df < ts
+        with pytest.raises(TypeError):
+            ts < df
+        with pytest.raises(TypeError):
+            ts > df
+
```

Comment:
can you add for tz as well (use the fixture)

Reply:
Sure; follow-up.  This PR is pretty specifically only-moving.